buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes.groovy

```diff
@@ -182,12 +182,14 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
                                         }
                                     }
 
-                                    // Cleanup zOS datasets 
+                                    // Cleanup zOS datasets
                                     if (nodeLabels.contains('sw.os.zos')) {
-                                        listcat = sh(script: "tso listcat | grep '${env.USER}' | grep 'JVM' | cut -d. -f 2-", returnStdout: true).trim()
-                                        listcat.split('\n').each {
-                                            sh "tso delete ${it}"
-                                        }
+                                        def listcat = sh(script: "tso listcat | grep '${env.USER}' | grep 'JVM' | cut -d. -f 2-", returnStdout: true).trim()
+                                            if (!listcat.isEmpty()) {
+                                                listcat.split('\n').each {
+                                                    sh "tso delete ${it}"
+                                                }
+                                            }
```

Comment:
Lines 188-192 are indented one level more than they should be.

Reply:
Made the change @keithc-ca